authentication-pre-request.js

```diff
@@ -0,0 +1,40 @@
+const { Header } = require('postman-collection');
+
+const { collectionVariables } = pm;
+
+const oauth2Request = {
+    url: 'https://api.oregonstate.edu/oauth2/token',
```

Comment:
Based on the [Airbnb style guide](https://github.com/airbnb/javascript#whitespace), the indentation should use 2 spaces.